packages/contracts-bedrock/src/L1/OptimismPortal2.sol

```diff
@@ -502,10 +534,18 @@ contract OptimismPortal2 is Initializable, ResourceMetering, ISemver {
         }
     }
 
+    /// @notice Migrates the total ETH balance to the ETHLockbox.
+    function migrateLiquidity() public {
+        if (msg.sender != PAO()) revert OptimismPortal_Unauthorized();
+        _migrateLiquidity();
+    }
+
     /// @notice Accepts deposits of ETH and data, and emits a TransactionDeposited event for use in
     ///         deriving deposit transactions. Note that if a deposit is made by a contract, its
     ///         address will be aliased when retrieved using `tx.origin` or `msg.sender`. Consider
     ///         using the CrossDomainMessenger contracts for a simpler developer experience.
+    /// @dev    The `msg.value` is locked on the ETHLockbox and minted as ETH when the deposit arrives on L2,
```

Comment:
We should really have this in forge fmt but we use max 100 line length for comments. I should at least add this to a linter or something. General nit, applies to whole PR. I'll make a ticket to get this enforced automatically or autoformatted.